kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_authentication.adoc

```diff
@@ -176,7 +176,7 @@ password.
 
 Some docker registries require additional steps to authenticate.
 link:https://docs.aws.amazon.com/AmazonECR/latest/userguide/ECR_GetStarted.html[Amazon ECR] requires using an IAM access key to obtain temporary docker login credentials.
-The <<jkube:push>> and <<jkube:build>> goals automatically execute this exchange for any registry of the form
+The {goal-prefix}:push and {goal-prefix}:build goals automatically execute this exchange for any registry of the form
```

Comment:
This removes the links/cross-references which actually print OK (reference title is printed) when the asciidoc is built.

Be very careful with this.

![image](https://user-images.githubusercontent.com/488391/122563065-0da1e780-d044-11eb-8a27-c74ec6ff6209.png)
![image](https://user-images.githubusercontent.com/488391/122563143-23171180-d044-11eb-96b4-a5c276e51580.png)